### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CodingSpecies**
@PulkitSinghDev @Panquesito7 , I tried to improve the readme.md, thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,21 +56,21 @@ get to collaborate with for your project, work, etc... & many more..... 🤩🤩
...
 
-We hope you enjoy it and get to learn something new and awesome from it
+## We hope you enjoy it and get to learn something new and awesome from it!
 
-& make sure to provide your reviews for this docs here:-
+## & make sure to provide your reviews for this docs here:-
 [Documentation_Reviews.md](https://github.com/PulkitSinghDev/OpenSource-for-Everyone/blob/main/Documentation_reviews.md)
 
-## Repo Admin / Maintainer
+## Repo Admin / Maintainer:
```
Comment (Panquesito7) [inline README.md:64]:
```suggestion
## Repository Administrator / Maintainer
```

File: README.md [hunk 2]
```diff
@@ -56,21 +56,21 @@ get to collaborate with for your project, work, etc... & many more..... 🤩🤩
...
 
 <a href="https://github.com/PulkitSinghDev"><img src="https://user-images.githubusercontent.com/71369943/115286432-ae6f4600-a16c-11eb-8139-2a6e8533651f.png" width="100px" height="100px"></a>
 
-## Resources
+## Resources:
```
Comment (Panquesito7) [inline README.md:68]:
```suggestion
## Resources
```

File: README.md [hunk 3]
```diff
@@ -56,21 +56,21 @@ get to collaborate with for your project, work, etc... & many more..... 🤩🤩
...
 
 - [LICENSE](LICENSE)
 - [Code of Conduct](CODE_OF_CONDUCT.md)
 - [OpenSource Guide](Guides)
 - [Learning Tracks](#learning-tracks-we-provide)
 
-# Support
+# Support:
```
Comment (Panquesito7) [inline README.md:75]:
```suggestion
# Support
```

**Comment by PulkitSinghDev** [comment]
sorry this commit is out dated @CodingSpecies , you can start contributing in this issue: [#issue 7](https://github.com/PulkitSinghDev/OpenSource-for-Everyone/issues/7)

**Review by PulkitSinghDev — COMMENTED** [review]
sorry this commit is outdated @CodingSpecies , you can start contributing in this issue: [#issue 7](https://github.com/PulkitSinghDev/OpenSource-for-Everyone/issues/7)

**Pull request closed without merging**